src/routes/index.js

```diff
@@ -8,10 +8,122 @@ const Password = require('../helpers/password')
 const { v4: uuidv4 } = require('uuid')
 const { ChatMessage, ChatMessageTable } = require('../entities/ChatMessage')
 
+const md = require('markdown-it')({
+  html: true,
+  typographer: true,
+  linkify: true,
+  highlight: (str, lang) =>
+    `
+        <code style="
+          display: block;
+          padding: 9.5px;
+          margin: 0px auto 0px auto;
+          font-size: 13px;
+          line-height: 1.42857143;
+          color: #333;
+          word-break: break-all;
+          word-wrap: break-word;
+          background-color: #f5f5f5;
+          border: 1px solid #ccc;
+          border-radius: 4px;"
+          >
+            ${str}
+        </code>
+    `,
+})
+
+const fs = require('fs')
+const path = require('path')
+const util = require('util')
+const user = require('../schemas/user')
+const chatMessage = require('../schemas/chatMessage')
+
+const readFile = util.promisify(fs.readFile)
+
 const routes = async (fastify) => {
+  // Docs
+  fastify.get(
+    '/ws/docs',
+    {
+      schema: {
+        description: 'Websocket docs at /ws/docs',
+      },
+    },
+    asyncHandler(async (request, reply) => {
+      const markdownOutput = await readFile(
+        path.resolve(__dirname, '../docs/ws.md'),
+        'utf8'
+      )
+
+      reply.type('text/html').send(md.render(markdownOutput))
+    })
+  )
+
+  // fastify.route({
```

Comment:
What's the purpose of the giant commented section?   If it has future relevance then leave it.